### ISSUE TITLE
Fail fast when accessibility isn't turned on on host machine

### ISSUE THREAD

**Issue by moredip** [moredip/Frank#20]
As a new user of Frank
I would like a clear error warning when my machine isn't configured correctly
So I can fix the problem and start rockin' it with Frank

Frank steps that use AppleScript to automate the Simulator fail in a fairly cryptic way when the machine running the steps doesn't have accessibility turned on in system preferences.

We should have those steps detect that they are failing because accessibility isn't turned on, and have them print a verbose, clear error message telling the user **why** they are having problems and **how to resolve them** by turning accessibility on.
